pokemon-minisite/src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,24 @@
+.header {
```

Comment:
please prettier this file.
you can use automatic prettier tool in vs code

Reply:
done